examples/Bickley_jet_shallow_water.jl

```diff
@@ -0,0 +1,258 @@
+# # Shallow Water Example: an unstable Bickley jet 
+#
+# This example shows how to use `Oceananigans.ShallowWwaterModel` to simulate
```

Comment:
`Oceananigans.ShallowWwaterModel` -> `Oceananigans.ShallowWaterModel`

[too many w's :)]

Reply:
Yes of course.  Sorry about that typo